dashboard/src/components/widget/loader/che-loader.directive.js

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2015-2016 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ */
+'use strict';
+
+/**
+ * Defines a directive for creating Loader
+ * @author Oleksii Kurinnyi
+ */
+export class CheLoader {
+
+  /**
+   * Default constructor that is using resource
+   * @ngInject for Dependency injection
+   */
+  constructor() {
+    this.restrict = 'E';
+    this.replace = true;
+    this.template = '<div ng-transclude class="loader"></div>';
```

Comment:
Is it good idea to create a new custom element for adding only one class "loader"? Instead of this we can prepared a class for us it with DOM elements. 
@ashumilova @benoitf  WDYT?